frontend/src/components/application-form.js

```diff
@@ -1,12 +1,59 @@
-/*
-This file contains all of the fields for the application form.  
-Each field is defined as an object with the following attributes:
-    label: the text that appears on the form beside the input
-    type: a string that determines whether the input is a default input field, a textbox, or a binary radio choice
-    field_name: the key name for the field
-    validate: possible validations, not yet implemented
-    required: whether or not the field is required to be filled
-*/
+import React from "react";
+
+export const DefaultInput = ({ label, curValue, onChange, required }) => (
+    <div>
+        <label style={{ width: "20vw" }}>
```

Comment:
Inline styles should be avoided. Use a `className` on the surrounding `div`

Reply:
Labels should surround their inputs or have an `htmlFor` attribute.

Reply:
The application form will be restyled and redesigned in a new issue.  Some of this code may be replaced.  I recommend we make changes in the new issue.